fuseki/sample-ubd.trig

```diff
@@ -206,7 +206,8 @@ graph <project/ubd/csv-mapping/table-observation> {
     cc:csvSource <cube-project/ubd/csv-source/ubd> ;
     schema:name "Observations" ;
     schema:color "#AAAAAA" ;
-    cc:identifierTemplate "ammonia/observation/{STATION}-{YEAR}-annualmean" ;
+    # cc:identifierTemplate "ammonia/observation/{STATION}-{YEAR}-annualmean" ;
+    cc:identifierTemplate "{station_id}-{year}-{aggregation_id}" ;
```

Comment:
Keeping the "correct" template around while the csvw is still fake and not built from all the mappings